package.json

```diff
@@ -21,6 +22,9 @@
     "type": "git",
     "url": "git+https://github.com/sigstore/sigstore-js.git"
   },
+  "publishConfig": {
+    "provenance": true
+  },
```

Comment:
The invocation of `npm publish` happens as part of the `changeset publish` [command](https://github.com/changesets/changesets/blob/main/packages/cli/src/commands/publish/npm-utils.ts#L177-L183) and doesn't give us an opportunity to inject additional flags (we should probably consider a PR to the changeset project with this change).

Since we can't pass the `--provenance` flag to the `publish` command directly we need to set it here (another option is to check-in a `.npmrc` file with this flag set). 

One nice side-effect of this approach is that it prevents anyone from accidentally publishing from their local machine (since you can't publish w/ provenance unless your running on GH Actions).

Reply:
I see now that this was already being discussed here: https://github.com/gh-community/npm-provenance-private-beta-community/issues/14

Reply:
Yes good idea using `publishConfig`! Getting lost in all the different ways you can configure npm.. 😄